models/ResumeModel.ts

```diff
@@ -12,7 +12,7 @@ export class ResumeModel extends BaseEntity {
   @Index('resumes_by_user_index')
   user: UserModel;
 
-  @Column('boolean', { default: false, nullable: false })
+  @Column('boolean', { default: true, nullable: false })
   isResumeVisible: boolean;
```

Comment:
Should we make this default true instead of false? I'd imagine we'd only want a user to have their resume viewable by recruiters if they specify they want to, otherwise we should hide by default. Let me know if you have any opinions on this though